src/content/reference/react-dom/flushSync.md

```diff
@@ -122,12 +122,12 @@ export default function PrintApp() {
 
 </Sandpack>
 
-Without `flushSync`, when the print dialog will display `isPrinting` as "no". This is because React batches the updates asynchronously and the print dialog is displayed before the state is updated.
+Tanpa `flushSync`, dialog cetak akan menampilkan `isPrinting` sebagai "no". Ini terjadi karena React menumpakkan pembaruan secara asinkron dan dialog cetak ditampilkan sebelum *state* diperbarui.
```

Comment:
To understand better.

```suggestion
Tanpa `flushSync`, dialog cetak akan menampilkan `isPrinting` sebagai "no". Ini terjadi karena React mengelompokkan (*batch*) pembaruan secara asinkron dan dialog cetak ditampilkan sebelum *state* diperbarui.
```

Reply:
I was referencing this glossary since it's not on this repo's glossary: http://kateglo.lostfocus.org/?phrase=batches&mod=glossary
Should I change it to mengelompokkan or keep it as it is?
Other possible translations:
- lompok (melompokkan)
- ruahan (meruahkan)

Reply:
@aris-uu We can change it to "mengelompokkan (*batch*)" for now. Since we're also gently introducing programming concepts in this guide, sometimes we need to switch our approach for loanwords.

See [this guideline](https://github.com/reactjs/id.react.dev/wiki/Panduan-Penulisan-Umum#penulisan-kata-serapan) for reference.